addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -343,8 +343,16 @@ const FieldEditor = FormEditor.extend({
     _getLabelPosition: function () {
         const label = this.$target[0].querySelector('.s_website_form_label');
         if (this.$target[0].querySelector('.row:not(.s_website_form_multiple)')) {
-            return label.classList.contains('text-end') ? 'right' : 'left';
+            if (label.classList.contains('text-end')) {
+                return 'right';
+            } else if (label.classList.contains('s_website_form_label_after')) {
+                return 'after';
+            }
+            return 'left';
         } else {
+            if (label.classList.contains('s_website_form_label_after')) {
+                return 'after';
+            }
             return label.classList.contains('d-none') ? 'none' : 'top';
```

Comment:
We may simplify this a bit 😉

```js
_getLabelPosition: function () {
    const label = this.$target[0].querySelector('.s_website_form_label');
    const isRow = this.$target[0].querySelector('.row:not(.s_website_form_multiple)');

    if (label.classList.contains('form-check-label')) {
        return 'after';
    }
    if (isRow) {
        return label.classList.contains('text-end') ? 'right' : 'left';
    }
    return label.classList.contains('d-none') ? 'none' : 'top';
},
```
note: to be tested. It also assumes that we can use `form-check-label` class.